recipes/openms-meta/meta.yaml

```
@@ -1,61 +1,84 @@
-      - yaml-cpp
-
-test:
+  # Shared build tools
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - pkg-config
+    - sysroot_linux-64 =2.17  # [linux]
+  # Shared host dependencies (used at link time)
+  host: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)